examples/react/src/pages/_app.tsx

```diff
@@ -1,23 +1,22 @@
 import { EthereumClient, modalConnectors, walletConnectProvider } from '@web3modal/ethereum'
-import { Web3Modal } from '@web3modal/react'
 import type { AppProps } from 'next/app'
-import { useRouter } from 'next/router'
 import { useEffect, useState } from 'react'
 import { configureChains, createClient, WagmiConfig } from 'wagmi'
 import { arbitrum, avalanche, bsc, fantom, mainnet, optimism, polygon } from 'wagmi/chains'
+import Navigation from '../components/Navigation'
 import '../styles.css'
 
 // 1. Get projectID at https://cloud.walletconnect.com
 if (!process.env.NEXT_PUBLIC_PROJECT_ID) {
   throw new Error('You need to provide NEXT_PUBLIC_PROJECT_ID env variable')
 }
 
-const projectId = process.env.NEXT_PUBLIC_PROJECT_ID
+export const projectId = process.env.NEXT_PUBLIC_PROJECT_ID
 
 // 2. Configure wagmi client
 const chains = [mainnet, polygon, optimism, arbitrum, avalanche, fantom, bsc]
 const { provider } = configureChains(chains, [walletConnectProvider({ projectId })])
-const wagmiClient = createClient({
+export const wagmiClient = createClient({
```

Comment:
NB: Understand this is being used in `custom.tsx `but would these `exports` be a bit confusing for developers looking at this example? MB if this is a weird nitpick.

Reply:
Yeh I agree, I think I will move wagmi initialisations etc... and have them defined in each page (need to do it anyway for wc v1 and v2 examples). Will be thinking on how to do this in a clean way on Monday. I think there might be conflicts in wamgi storage if I do it this way